convert2rhel/subscription.py

```diff
@@ -469,7 +431,10 @@ def __call__(self):
 
             try:
                 if self.password:
-                    loggerinst.info("Registering via username/password: %s" % " ".join(utils.hide_secrets(self.args)))
+                    loggerinst.info("Organization: %s", self.org)
```

Comment:
I'm a bit unsure if we have to print `Organization:` here... Maybe we do because we set that in the line 439 :thinking:  I guess it would be nice to confirm with @abadger since he did the code a while ago.

Reply:
Also, if we do, I would do conditional to print that only if there is an org present otherwise, I would omit this information as it is not relevant if it's not present, like so: 

```suggestion
                    if self.org:
                        loggerinst.info("Organization: %s", self.org)

```

Reply:
The indentation may be wrong, though...

Reply:
I'll wait on @abadger to respond for this change

Reply:
It turns out that organization is needed even for username and password if the username is associated with more than one organization.  We have a ticket open to handle that case: https://issues.redhat.com/browse/RHELC-726

So logging the organization is a good idea here.